.github/workflows/master.yml

```diff
@@ -23,6 +23,19 @@ jobs:
 
     steps:
     - uses: actions/checkout@master
+    # We split caches because GitHub Action Cache has a 400MB-size limit.
+    - uses: actions/cache@v1
+      with:
+        path: ~/.m2/repository/com
+        key: ${{ matrix.java }}-${{ matrix.hadoop }}-maven-com-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ matrix.java }}-${{ matrix.hadoop }}-maven-com-
+    - uses: actions/cache@v1
+      with:
+        path: ~/.m2/repository/org
+        key: ${{ matrix.java }}-${{ matrix.hadoop }}-maven-org-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ matrix.java }}-${{ matrix.hadoop }}-maven-org-
```

Comment:
How about those that are not in `com` or `org`?

Reply:
It will be handled in the same manner we used before.
They are too many files with small sizes and the flakiness occurs for the above two directories until now.

Reply:
We can put them together when GitHub Action increase the limit (400MB) later.

Reply:
Yeah, seems they are going to increase the limit with zstd codec.